packages/contracts-bedrock/test/kontrol/pausability-lemmas.md

```diff
@@ -0,0 +1,234 @@
+Kontrol Lemmas
```

Comment:
_(arbitrary comment location to start a thread)_

The README still references `pausability-lemmas.k` twice, we'll have to update that

Reply:
Updated it accordingly :ok_hand: